domains/libraries/EnergyOrigin.Domain/EnergyOrigin.Domain.ValueObjects/Converters/UnixTimestampValueConverter.cs

```diff
@@ -4,7 +4,9 @@ namespace EnergyOrigin.Domain.ValueObjects.Converters;
 
 public class UnixTimestampValueToSecondsConverter() : ValueConverter<UnixTimestamp, long>(v => v.EpochSeconds, v => UnixTimestamp.Create(v));
 
+public class UnixTimestampValueToDateTimeOffsetConverter()
```

Comment:
Many classes one file. Not sure how I feel. Maybe class name be more generic? DatabaseValueConverters.